src/components/media/MediaMenu.tsx

```diff
@@ -33,19 +42,25 @@ const MediaMenu = ({
     isDisabled = false,
     dropdownProps = {},
     menuProps = {},
+    intl,
     ...rest
-}: MediaMenuProps) => (
-    <DropdownMenu constrainToScrollParent isRightAligned {...dropdownProps}>
-        <PlainButton
-            isDisabled={isDisabled}
-            className={classnames('bdl-Media-menu', className)}
-            type={ButtonType.BUTTON}
-            {...rest}
-        >
-            <IconEllipsis color={bdlGray50} height={16} width={16} />
-        </PlainButton>
-        <Menu {...menuProps}>{children}</Menu>
-    </DropdownMenu>
-);
-
-export default MediaMenu;
+}: MediaMenuProps) => {
+    const commentOptionsAriaLabel = intl.formatMessage(messages.commentOptionsArialLabel);
```

Comment:
Can we inline this call with `aria-label` to allow this component to remain an implicit return function?